docs/content/preview/yugabyte-cloud/release-notes.md

```diff
@@ -19,6 +19,14 @@ On this page:
 
 ## Change log
 
+### June 24, 2024
+
+**New Features**
+
+- YugabyteDB Managed is now YugabyteDB Aeon! [Learn more](https://www.yugabyte.com/blog/introducing-yugabytedb-aeon/).
+- Support for new pricing plans on clusters. Existing customers continue to be billed using classic pricing. [Learn more](https://www.yugabyte.com/pricing/).
```

Comment:
@ddhodge Change to "Support for new pricing plans" as it applies to accounts, not clusters.